### PR TITLE
Create test-snap for snap_dialog

### PR DESCRIPTION
**Pull request by bowensanders**
This creates a test-snaps panel and snap for testing snap_dialog

### CODE REVIEW

File: packages/dialog/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "test": "jest --passWithNoTests",
+    "test:watch": "jest --watch",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --single-quote --ignore-path ../../.prettierignore --no-error-on-unmatched-pattern",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:changelog": "yarn auto-changelog validate"
+  },
+  "dependencies": {
+    "@metamask/snaps-types": "^0.26.2"
```
Comment (FrederikBolding) [inline packages/dialog/package.json:35]: Add snap UI here

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: packages/dialog/package.json (hunk 1 above)
Comment (bowensanders) [reply packages/dialog/package.json:35]: `    "@metamask/snaps-ui": "^0.27.1",` is on line 44 already

Comment (FrederikBolding) [reply packages/dialog/package.json:35]: Ah! Then it needs to be moved to be a regular dependency instead of a dev dependency

Comment (bowensanders) [reply packages/dialog/package.json:35]: done, fix pushing!

File: packages/dialog/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [4.4.1]
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/dialog/CHANGELOG.md:10]: I believe this has been fixed now

[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/dialog/CHANGELOG.md]: yes this has indeed now been fixed. :)

File: packages/dialog/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
```
Comment (FrederikBolding) [inline packages/dialog/CHANGELOG.md:8]:
```suggestion
```

File: packages/dialog/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+- Created snap_dialog test panel and snap
```
Comment (FrederikBolding) [inline packages/dialog/CHANGELOG.md:9]:
```suggestion
```

File: packages/dialog/CHANGELOG.md (hunk 3 above)
Comment (FrederikBolding) [reply packages/dialog/CHANGELOG.md:8]: These will be added automatically when we create the release

File: packages/dialog/CHANGELOG.md (hunk 4 above)
Comment (FrederikBolding) [reply packages/dialog/CHANGELOG.md:9]: These will be added automatically when we create the release

File: packages/dialog/CHANGELOG.md (hunk 3 above)
Comment (bowensanders) [reply packages/dialog/CHANGELOG.md]: Done :)

File: packages/dialog/CHANGELOG.md (hunk 4 above)
Comment (bowensanders) [reply packages/dialog/CHANGELOG.md]: Done

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**